js/controller/play.js

```diff
@@ -43,7 +43,8 @@ angular.module('listenone').controller('PlayController', [
     $scope.currentPosition = '0:00';
 
     if (!$scope.isChrome) {
-      $scope.isMac = require('electron').remote.process.platform === 'darwin';
+      // eslint-disable-next-line no-undef
```

Comment:
any reason?

Reply:
I think we can directly access `process` without the use of remote.
Just open the console on desktop version and type `process.platform` to confirm.

Reply:
Since some code is not tested though multiple os systems, I'm not sure about the side effect of changes without carefully tested. So maybe another pr for deprecate remote module?

Reply:
https://www.electronjs.org/docs/api/process
According to doc the api is available both in the main and renderer